sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/indexes/models/SearchServiceCounters.java

```diff
@@ -56,31 +56,45 @@ public final class SearchServiceCounters implements JsonSerializable<SearchServi
     /*
      * Total number of skillsets.
      */
-    private ResourceCounter skillsetCounter;
+    private final ResourceCounter skillsetCounter;
+
+    /*
+     * Total memory consumption of all vector indexes within the service, in bytes.
+     */
+    private final ResourceCounter vectorIndexSizeCounter;
 
     /**
      * Creates an instance of SearchServiceCounters class.
      *
+     * @param aliasCounter the aliasCounter value to set.
      * @param documentCounter the documentCounter value to set.
      * @param indexCounter the indexCounter value to set.
      * @param indexerCounter the indexerCounter value to set.
      * @param dataSourceCounter the dataSourceCounter value to set.
      * @param storageSizeCounter the storageSizeCounter value to set.
      * @param synonymMapCounter the synonymMapCounter value to set.
+     * @param skillsetCounter the skillsetCounter value to set.
+     * @param vectorIndexSizeCounter the vectorIndexSizeCounter value to set.
      */
     public SearchServiceCounters(
+            ResourceCounter aliasCounter,
```

Comment:
We need to use a Swagger transform on this class as adding new required fields breaks the public API. `aliasCounter`, `skillsetCounter`, and `vectorIndexSizeCounter` will need to be made not required.

Reply:
@alzimmermsft I also noticed that the regeneration removed the `@Fluent` annotation and added the `@Immutable` annotation. Would this not break anything? I feel like it could potentially but I'm not certain